index.js

```diff
@@ -13,15 +13,55 @@ const pool = new Pool({
 
 app.get('/api/generate-deck', (req, res) => {
     const requestedDeck = createDeck();
+
+    try {
+        const insertDeckInfo = `
+            INSERT INTO
+                decks (id, name, set, house1, house2, house3)
+            VALUES
+                (DEFAULT, $1, $2, $3, $4, $5
+            ON CONFLICT (name)
+            DO NOTHING
+            RETURNING id
+        `;
+        const deckInsertResponse = await pool.query(insertDeckInfo, [requestedDeck.name, requestedDeck.expansion, requestedDeck.houses[0], requestedDeck.houses[1], requestedDeck.houses[2]]);
```

Comment:
`requestedDeck.expansion` is a property of a card, not a deck if I remember correctly